template/deployments/kustomize/base/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap
+data:
```

Comment:
same here, you ahve to have a reference to the draft variable for the envvariables here